### ISSUE TITLE
Carthage build fails for 2.0.8, 2.0.9

### ISSUE THREAD

**Issue by danqing** [tilltue/TLPhotoPicker#258]
2.0.7 works alright.

Relevant error:

```
.../Carthage/Checkouts/TLPhotoPicker/TLPhotoPicker/Classes/TLPhotosPickerViewController.swift:1157:33: error: use of unresolved identifier 'TLAssetPreviewViewController'
        let previewController = TLAssetPreviewViewController()
                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
.../Carthage/Checkouts/TLPhotoPicker/TLPhotoPicker/Classes/TLPhotosPickerViewController.swift:1171:41: error: use of unresolved identifier 'TLAssetPreviewViewController'
                let previewController = TLAssetPreviewViewController()
                                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~

** ARCHIVE FAILED **
```

**Comment by intoxicated** [comment]
https://github.com/tilltue/TLPhotoPicker/pull/259 I made PR for it

**Comment by intoxicated** [comment]
@danqing #259 merged and told that 2.0.10 was released.

**Comment by danqing** [comment]
Worked great.
